common/testData/localeIdentifiers/likelySubtags.txt

```diff
@@ -802,11 +812,6 @@ pl-AQ ;	pl-Latn-AQ ;	pl-AQ ;
 pl-Egyp ;	pl-Egyp-PL ;	pl-Egyp ;	
 pl-Latn ;	pl-Latn-PL ;	pl ;	
 pl-PL ;	pl-Latn-PL ;	pl ;	
-prg ;	prg-Latn-PL ;	prg ;	
```

Comment:
Why is prg being removed? It doesn't make sense to me that it dropped coverage between CLDR 46 and 47 since it was only at Basic and AFAIK nothing changed in Basic in 47. Do you know what happened?

Reply:
I would like to go ahead and merge this; it shouldn't be a blocker for the alpha

Reply:
Okay, approved and we can investigate the change in coverage elsewhere.